### PR TITLE
feat: add param SmallLiquidationSize

### PR DESCRIPTION
**Pull request by toteki**
## Description

If a borrower's borrowed value (in USD) is lower than this `sdk.Dec` parameter, their `CloseFactor` will be `1.0` once they are eligible for liquidation.

Thus, "small" borrows abandoned by their users are not chipped away at forever by ~1% MinimumCloseFactor liquidations, never reaching zero, but can instead by finished off.

The default SmallLiquidationSize is $100.00

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] added appropriate labels to the PR
- [ ] added `!` to the type prefix if API or client breaking change
- [x] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/leverage/spec/07_params.md [hunk 1]
```diff
@@ -7,19 +7,25 @@ The leverage module contains the following parameters:
...
 
 OracleRewardFactor is the portion of borrow interest accrued that goes to fund
-the `x/oracle` reward pool.
\ No newline at end of file
+the `x/oracle` reward pool.
+
+## SmallLiquidationSize
+
+SmallLiquidationSize is the borrow value in USD below which [Close Factor](01_concepts.md#Close-Factor)
+is always 1.0 once the borrow is eligible for liquidation.
```
Comment (toteki) [inline x/leverage/spec/07_params.md:31]:
```suggestion
is always 1.
```

**Review by RafilxTenfen — APPROVED** [review]

**Comment by RafilxTenfen** [comment]
Weird, protobuf is also failing at main

cc @adamewozniak @toteki

**Comment by adamewozniak** [comment]
> Weird, protobuf is also failing at main
> 
> 
> 
> cc @adamewozniak @toteki 

Afaik that test has been failing if there are any protobuff changes for a while - we should definitely fix it, but shouldn't be a problem in this PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+- [723](https://github.com/umee-network/umee/pull/723) Add leverage parameter SmallLiquidationSize
```
Comment (adamewozniak) [inline CHANGELOG.md:51]:
```suggestion
- [723](https://github.com/umee-network/umee/pull/723) Add leverage parameter SmallLiquidationSize, which determines the USD value at which a borrow is considered small enough to be liquidated in a single transaction.
```

**Pull request merged**